### PR TITLE
refactor: update scripts for fork users

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

When people fork this repository and add their private repositories, they have to fix some parts such as:
- Change the default playbook name.
- Add a token to clone private repositories.

To make it easier, I'll refactor the code.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -15,7 +15,12 @@ jobs:
 
       - name: Run setup script
         run: |
-          ./setup-dev-env.sh -y universe
+          ./setup-dev-env.sh -y
+
+      - name: Set git config
+        uses: autowarefoundation/autoware-github-actions/set-git-config@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kenji-miyake) [inline .github/workflows/build.yaml:23]: The fork users will change this to `secrets.REPO_TOKEN`.

File: .github/workflows/github-release.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ on:
   workflow_dispatch:
     inputs:
       beta-branch-or-tag-name:
+        description: The name of the beta branch or tag to release
```
Comment (kenji-miyake) [inline .github/workflows/github-release.yaml:12]: This is just a follow-up for linting.

File: setup-dev-env.sh [hunk 3]
```diff
@@ -83,6 +83,12 @@ if ! (command -v sudo >/dev/null 2>&1); then
     apt-get -y install sudo
 fi
 
+# Install git
+if ! (command -v git >/dev/null 2>&1); then
+    sudo apt-get -y update
+    sudo apt-get -y install git
+fi
```
Comment (kenji-miyake) [inline setup-dev-env.sh:90]: `set-git-config` requires `git`.

**Comment by kenji-miyake** [comment]
Build CI: https://github.com/autowarefoundation/autoware/runs/6433565563?check_suite_focus=true

**Review by esteve — APPROVED** [review]

**Pull request merged**